Ensconce/Program.cs

```diff
@@ -287,6 +289,11 @@ private static void SetUpAndParseOptions(string[] args)
                                 s => withTransaction = Convert.ToBoolean(s)
                             },
                             {
+                                "roundhouseOutputPath=",
```

Comment:
@richard-green, as you've altered the commandline you should probably update the [Wiki](https://github.com/15below/Ensconce/wiki) as well.